src/sources/opentelemetry/integration_tests.rs

```diff
@@ -182,6 +187,108 @@ async fn receive_trace() {
     .await;
 }
 
+#[tokio::test]
+async fn receive_metric() {
+    // generate a metrics gauge request
+    let req = ExportMetricsServiceRequest {
+        resource_metrics: vec![ResourceMetrics {
+            resource: Some(Resource {
+                attributes: vec![KeyValue {
+                    key: "service.name".to_string(),
+                    value: Some(AnyValue {
+                        value: Some(StringValue("vector-collector".to_string())),
+                    }),
+                }],
+                dropped_attributes_count: 0,
+            }),
+            schema_url: "".to_string(),
+            scope_metrics: vec![ScopeMetrics {
+                scope: Some(InstrumentationScope {
+                    name: "vector-collector-instrumentation".to_string(),
+                    version: "0.111.0".to_string(),
+                    attributes: vec![],
+                    dropped_attributes_count: 0,
+                }),
+                schema_url: "".to_string(),
+                metrics: vec![Metric {
+                    name: "some.random.metric".to_string(),
+                    description: "Some random metric we use for test".to_string(),
```

Comment:
```suggestion
                    description: "Some random metric we use for test".to_string(),
```
Just want to be sure this is the intended description here ?

Reply:
I synced with @pront on this and understand this is not user facing content so I will dismiss this review, sorry for the confusion !